Fix new flake8 error

Impacts `master`
https://travis-ci.org/pytorch/ignite/jobs/486833029#L466
```
$ flake8
./ignite/metrics/accuracy.py:42:17: E117 over-indented
./tests/ignite/metrics/test_metrics_lambda.py:11:9: E117 over-indented
```

and `nightly`
https://travis-ci.org/pytorch/ignite/jobs/486949548#L452

--- a/.travis.yml
+++ b/.travis.yml
@@ -27,7 +27,7 @@ before_install: &before_install
   - conda info -a
   - conda create -q -n test-environment -c pytorch python=$TRAVIS_PYTHON_VERSION $PYTORCH_PACKAGE
   - source activate test-environment
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then pip install enum34; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then pip install enum34 typing; fi
   # Test contrib dependencies
   - pip install tqdm scikit-learn
 

--- a/ignite/metrics/accuracy.py
+++ b/ignite/metrics/accuracy.py
@@ -39,7 +39,7 @@ class _BaseClassification(Metric):
             raise ValueError("y and y_pred must have compatible shapes.")
 
         if self._is_multilabel and not (y.shape == y_pred.shape and y.ndimension() > 1 and y.shape[1] != 1):
-                raise ValueError("y and y_pred must have same shape of (batch_size, num_categories, ...).")
+            raise ValueError("y and y_pred must have same shape of (batch_size, num_categories, ...).")
 
         return y_pred, y
 

--- a/tests/ignite/metrics/test_metrics_lambda.py
+++ b/tests/ignite/metrics/test_metrics_lambda.py
@@ -8,18 +8,18 @@ import torch
 
 class ListGatherMetric(Metric):
 
-        def __init__(self, index):
-            super(ListGatherMetric, self).__init__()
-            self.index = index
+    def __init__(self, index):
+        super(ListGatherMetric, self).__init__()
+        self.index = index
 
-        def reset(self):
-            self.list_ = None
+    def reset(self):
+        self.list_ = None
 
-        def update(self, output):
-            self.list_ = output
+    def update(self, output):
+        self.list_ = output
 
-        def compute(self):
-            return self.list_[self.index]
+    def compute(self):
+        return self.list_[self.index]
 
 
 def test_metrics_lambda():

